src/main/common/filter.h

```diff
@@ -75,6 +75,12 @@ typedef enum {
     FILTER_BPF,
 } biquadFilterType_e;
 
+typedef enum {
+    STAGE2_FILTER_NONE = 0,
+    STAGE2_FILTER_BIQUAD_RC_FIR2,
+    STAGE2_FILTER_FKF,
```

Comment:
Since this is not a Kalman filter, I think it should be renamed into something like `STAGE2_FILTER_RF` or similar. Using the term 'FKF' here is just confusing.